src/Mentat.Domain/Utils/Enums.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Mentat.Domain.Utils
+{
+    public enum DifficultyLevel
+    {
+        Easy,
+        Medium,
+        Hard,
+        Expert
+    }
```

Comment:
Since I wouldn't consider this a utility I think this would be a better fit in the Constants.cs file. Also, it would be easier to work with as a public static string[], but I could see it either way.

```
public static string [] FlashCardDifficultyLevels = new string [] { ...}
```

Reply:
Constants.cs probably is the best place for this, or maybe even just an Enums.cs file at the same hierarchy level as the constants file.

I don't have particularly strong feelings about enum versus a string array, though I guess I should mention that under the hood, C# enums are converted to integers (the enum name is just a label for the number). Both approaches have their merits and drawbacks. Given that we don't have plans (at this time, anyway) to manipulate the values of a DifficultyLevel programmatically, I guess it's really up to you (Rachel) which path to take.

Reply:
@dyohn I originally had the file at the same level as Constants.cs, but I was having trouble referencing it in other layers, which is why I added the folder. I can try again though.

Reply:
Interesting. You might be able to get it imported with a statement like `using static Mentat.Domain.Constants` or `using static Mentat.Domain.Enums`. That's a handy little trick for getting static methods/fields. Assume `Enums.cs` is a static class with a static method `GetEnumIntegerValue(string enumLable)`, the approach I described would allow you to invoke it as `GetEnumIntegerValue(whatever)` instead of `Enums.GetEnumIntegerValue(whatever)`. I use this sometimes for helper/logger functions.

Reply:
@dyohn I moved it to Constants.cs and `@using static Mentat.Domain.Constants` worked. Thank you much!

Reply:
Committed the changes. Resolving.